packages/vaadin-lumo-styles/src/mixins/base-layer-reset.css

```diff
@@ -0,0 +1,7 @@
+@layer base {
+  :is(:host, *, #dummy) {
```

Comment:
I’m not sure "dummy" is the ID we want to use in the final code 😄 It was just the first thing that crossed my mind when thinking of something that is there only to bump the specificity. Perhaps just `#id`?